src/GetOrganized/GetOrganizedService.php

```diff
@@ -205,11 +230,18 @@ private function getOrganizedDocuments()
 
     private function client()
     {
+        $httpClient = new TraceableHttpClient(Client::createHttpClient(
```

Comment:
To avoid making an unused client after one has already been made, this could be moved into the following `if` construct.